packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.js

```diff
@@ -27,15 +36,22 @@ function SingleImageViewer(props) {
   return (
     <SVGContext.Provider value={{ canvas }}>
       {zoomControlFn && (
-        <ZoomControlButton onClick={zoomControlFn} zooming={zooming} />
+        <ZoomControlButton
+          onClick={zoomControlFn}
+          zooming={zooming}
+        />
       )}
-      <Box animation='fadeIn' overflow='hidden'>
+      <StyledBox
+        animation='fadeIn'
+        invert={invert}
+        overflow='hidden'
+      >
         <svg
           focusable
```

Comment:
```suggestion
        <svg
          filter={invert ? 'invert(100%)' : 'invert(0)'}
          focusable
```

Using a style attribute on the SVG element here should fix the invert button in Safari.